neurom/fst/tests/test_bifurcationfunc.py

```diff
@@ -26,58 +26,11 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-'''Test neurom._bifurcationfunc functionality'''
+'''Test neurom._neuritefunc functionality'''
 
-import os
-import warnings
+from neurom.features.tests.test_bifurcationfunc import *
 
-import numpy as np
-from nose import tools as nt
-from nose.tools import assert_equal, assert_raises
-from numpy.testing import assert_raises
-
-import neurom as nm
-from neurom import load_neuron
-from neurom.exceptions import NeuroMError
-from neurom.fst import _bifurcationfunc as bf
-
-_PWD = os.path.dirname(os.path.abspath(__file__))
-DATA_PATH = os.path.join(_PWD, '../../../test_data/')
-SWC_PATH = os.path.join(DATA_PATH, 'swc')
-SIMPLE = nm.load_neuron(os.path.join(SWC_PATH, 'simple.swc'))
-with warnings.catch_warnings(record=True):
-    SIMPLE2 = load_neuron(os.path.join(DATA_PATH, 'neurolucida', 'not_too_complex.asc'))
-    MULTIFURCATION = load_neuron(os.path.join(DATA_PATH, 'neurolucida', 'multifurcation.asc'))
-
-
-def test_local_bifurcation_angle():
-    nt.ok_(bf.local_bifurcation_angle(SIMPLE.sections[1]) == np.pi)
-    nt.ok_(bf.local_bifurcation_angle(SIMPLE.sections[4]) == np.pi)
-    assert_raises(NeuroMError, bf.local_bifurcation_angle, SIMPLE.sections[0])
-
-def test_remote_bifurcation_angle():
-    nt.ok_(bf.remote_bifurcation_angle(SIMPLE.sections[1]) == np.pi)
-    nt.ok_(bf.remote_bifurcation_angle(SIMPLE.sections[4]) == np.pi)
-    assert_raises(NeuroMError, bf.local_bifurcation_angle, SIMPLE.sections[0])
-
-def test_bifurcation_partition():
-    root = SIMPLE2.neurites[0].root_node
-    assert_equal(bf.bifurcation_partition(root), 3.0)
-    assert_equal(bf.bifurcation_partition(root.children[0]), 1.0)
-
-    leaf = root.children[0].children[0]
-    assert_raises(NeuroMError, bf.bifurcation_partition, leaf)
-
-    multifurcation_section = MULTIFURCATION.neurites[0].root_node.children[0]
-    assert_raises(NeuroMError, bf.bifurcation_partition, multifurcation_section)
-
-def test_partition_asymmetry():
-    root = SIMPLE2.neurites[0].root_node
-    assert_equal(bf.partition_asymmetry(root), 0.5)
-    assert_equal(bf.partition_asymmetry(root.children[0]), 0.0)
-
-    leaf = root.children[0].children[0]
-    assert_raises(NeuroMError, bf.partition_asymmetry, leaf)
-
-    multifurcation_section = MULTIFURCATION.neurites[0].root_node.children[0]
-    assert_raises(NeuroMError, bf.partition_asymmetry, multifurcation_section)
+from neurom.fst import _bifurcationfunc as bifurcation
```

Comment:
Could you:
a) have a comment explaining how this override works (in all places it's used)
b) in `neurom.features.tests.test_bifurcationfunc` point to all the files that need to be updated if one of the names that are being over-written here changes - w/ a TODO to say remove it when these tests go away.